### ISSUE TITLE
navigating from Java to HTML does not work if there is an space between ( and string 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bwebb3856/qwickie#49]
```
How to see this?

1- new Form("a") works ok.
2- new Form( "a" ) does not works.

I will try to provide patch... 
```

Original issue reported on code.google.com by `reier...@gmail.com` on 17 Apr 2013 at 6:27

**Comment by GoogleCodeExporter** [comment]
```
Yes, I know some time that will come...
```

Original comment by `count.negative` on 17 Apr 2013 at 7:04
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Shall I look at the issue myself, my boss kindly allows me to spend some time 
helping fix the issue. Or you will take care of it? I was planning to look at 
package

https://code.google.com/p/qwickie/source/browse/#svn%2Ftrunk%2Fqwickie.plugin%2F
src%2Fqwickie%2Fhyperlink

By the way, thanks for the nice plugin:-)


```

Original comment by `reier...@gmail.com` on 17 Apr 2013 at 7:18

**Comment by GoogleCodeExporter** [comment]
```
It's in the DocumentHelper.findWord function. Help will be appreciated cause I 
require more time for that (and I don't have such a kind boss)
```

Original comment by `count.negative` on 17 Apr 2013 at 7:21

**Comment by GoogleCodeExporter** [comment]
```
Ok. I will have a look.

P.S. Boss contributes himself to OpenSource so he knows that it all boils down 
to.
```

Original comment by `reier...@gmail.com` on 17 Apr 2013 at 7:27

**Comment by GoogleCodeExporter** [comment]
```
nah, stop. It's in findWicketComponentRegion where I try to find if the 
wicket:id is the first param in a call. Line 273. Gosh there are testcases but 
not for blanks. And think of generics like: 
final Form<Customer> form = new Form<Customer>(   "customer", customerModel);
```

Original comment by `count.negative` on 17 Apr 2013 at 7:36

**Comment by GoogleCodeExporter** [comment]
```
ok, now I tried to ignore the blanks in line 264 (DocumentHelper)
                if (c == '(' /*|| c == ' '*/) {

got to test a bit.

```

Original comment by `count.negative` on 17 Apr 2013 at 8:01

**Comment by GoogleCodeExporter** [comment]
```
checked in my changes. Could you please do some testing? Releasing is only 
possible at home cause of our proxy here at work.

Thanks for your bug report and your time!
```

Original comment by `count.negative` on 17 Apr 2013 at 8:19

**Comment by GoogleCodeExporter** [comment]
```
Thanks to your for fixing this issue! I have tested it locally and it seems to 
work fine. I attach some test cases for the tests project.

We would appreciate if you can do a release any time soon.

Thanks again!

Cheers,

Ernesto

P.S. Greetings from my boss
```

Original comment by `reier...@gmail.com` on 17 Apr 2013 at 11:21

**Comment by GoogleCodeExporter** [comment]
```
test with generics too 
```

Original comment by `reier...@gmail.com` on 17 Apr 2013 at 11:27

Attachments:
- [issue49-tests.patch](https://storage.googleapis.com/google-code-attachments/qwickie/issue-49/comment-9/issue49-tests.patch)

**Comment by GoogleCodeExporter** [comment]
```
Released 1.1.1
```

Original comment by `count.negative` on 17 Apr 2013 at 4:49
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks!

Cheers,

Ernesto
```

Original comment by `reier...@gmail.com` on 17 Apr 2013 at 4:55
